### PR TITLE
Add LocationIQ provider

### PR DESCRIPTION
**Pull request by SrihariThalla**

### CODE REVIEW

File: src/Provider/LocationIQ/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Change Log
+
+The change log describes what is "Added", "Removed", "Changed" or "Fixed" between each release.
+
+## 4.1.0
```
Comment (Nyholm) [inline src/Provider/LocationIQ/CHANGELOG.md]: Use 1.0

File: src/Provider/LocationIQ/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2017 — Srihari Thalla <srihari@unwiredlabs.com>
```
Comment (Nyholm) [inline src/Provider/LocationIQ/LICENSE]: You need to use the license file from the root package

**Comment by SrihariThalla** [comment]
Hi, @Nyholm I've updated the requested changes. Can you look into it? Thanks!

**Comment by Nyholm** [comment]
The LocaitonIQ has a repo https://github.com/geocoder-php/locationiq-provider and a packagist package now.

**Pull request merged**